docs/scribble/components/pio/Overview.jinja2

```diff
@@ -0,0 +1,4 @@
+= Parallel I/O (PIO)
+
+The {{ product_name }} contains {{ devices|length|english_number }} parallel I/O (PIO) block{{ devices|pluralize(" is,s are")}} for simple parallel input/output.
```

Comment:
I think you want to get rid of the is/are

Reply:
Good catch. Fixed in https://github.com/sifive/block-pio-sifive/pull/24/commits/d3d2925b27ff4d0839f202ba9f3c75d5745e7648.